nvflare/private/fed/app/fl_conf.py

```diff
@@ -230,6 +240,7 @@ def __init__(
         )
 
         self.components = {}  # id => component
+        self.handlers = []
```

Comment:
what are these handlers for?

Do we have an example of how these are used?

Reply:
This is to enable client side also define the components in fed_client.json. We don't have the exact usage yet.